src/views/blocks/rsvp/form/opt-out.php

```diff
@@ -14,28 +14,40 @@
  * @version 4.9.4
```

Comment:
Are we changing the RSVP block for the modal?

Reply:
My understanding is no - we're not changing the RSVP block yet.

Reply:
So this view change here is just for compatibility with our other changes? Asking since there are class changes as well which could change appearance?

Reply:
Yes, I changed the class in one place, so I changed it everywhere - it should not affect the RSVP view.